main_app/templates/main_app/helpline_numbers.html

```diff
@@ -208,9 +212,1025 @@ <h1 class="wb-stl-heading1">
                                     aria-hidden="true"></i></a></span></strong></p>
                     </td>
                 </tr>
-
             </tbody>
         </table>
+        
+
+
+        
+            
+            
+
+    </div>
+
+
+    
+        
+    <div class="column" style="margin-top: 50px; text-shadow: 2px 2px grey; text-align: center; margin-bottom: 30px;">
+        <h1 >
+            <strong>
+                <font color="black">WOMEN HELPLINE NUMBERS STATEWISE</font>
+            </strong>
+            <img src="{% static 'Images/helpline.png' %}" alt="24 * 7" height="50vw">
+        </h1>
+    </div>
+    
+
+    <div class="container">
+            
+        <ul class="collapsible popout" data-collapsible="accordion">
+            <li>
+                <div class="collapsible-header">
+                    <i class="material-icons">arrow_drop_down</i>
+                    <p style="height: 30px; padding-top: 35px;" width="275" class="wb-stl-normal"><span class="wb-stl-special"><span> &nbsp; &nbsp; <h3><strong>DELHI</strong></h3></span></span></p>
+                </div>
+                <div class="collapsible-body">
+                    <table class="table table-striped " cellpadding="20" cellspacing="4" style="background-color:white;color:black;font-family:lobster,lobster,sans-serif;font-size:3vw; width:100%; " width="1250px">
+                        <tbody class="womenhp2">
+                            <tr>
+                                <td style="height: 100px; padding-top: 35px;" width="275px">
+                                    <p class="wb-stl-normal"><span class="wb-stl-special"><span>&#8226; &nbsp; &nbsp;Delhi Commison for women</span></span></p>
```

Comment:
- [ ] Change the font style of all lists. Make it similar to the font style of `Women Helpline - All India`.
- [ ] The font size must be similar to the `Women Helpline - All India`.

Reply:
- [x] Change the font style of all the headings. Make it similar to the font style of  `Women Helpline - All India.`
- [x] Add hovering effect to the block of headings.
- [x] Change the font style of all lists. Make it similar to the font style of `Women Helpline - All India.`
- [ ] The font size must be similar to the `Women Helpline - All India.`

@sakshikhachane Actually the size of list is made small to had a difference in header and list.
This thing is suggested by @swapnilsparsh. If you wish i can revert back the changes. 

Please have a look on the same and let me know for further correction if any.
![1](https://user-images.githubusercontent.com/67692721/115898693-783b0a80-a47b-11eb-892f-be7ffecf7665.JPG)
![2](https://user-images.githubusercontent.com/67692721/115898707-7bce9180-a47b-11eb-9dbb-400902ac7c2e.JPG)